groups/sig-cli/groups.yaml

```
@@ -7,6 +7,18 @@ groups:
   # and is not intended to govern access to infrastructure
   #
 
+  - email-id: k8s-sig-cli-leads@kubernetes.io
+    name: k8s-sig-cli-leads
+    description: |-
+      SIG CLI leads list
+    settings:
+      ReconcileMembers: "true"
```

```suggestion
    settings:
      WhoCanPostMessage: "ANYONE_CAN_POST"
      ReconcileMembers: "true"
```

Just so you can get messages from zoom etc.